tools/hyphy/macros.xml

```diff
@@ -31,11 +31,38 @@
         </param>
     </xml>
 
+    <xml name="conditional_posteriorEstimationMethod">
+        <conditional name="posteriorEstimationMethod">
+            <param argument="--method" type="select" label="Posterior estimation method">
+                <option value="Variational-Bayes">0-th order Variational Bayes approximation</option>
+                <option value="Metropolis-Hastings">Full Metropolis-Hastings MCMC algorithm</option>
+                <option value="Collapsed-Gibbs">Collapsed Gibbs sampler</option>
+            </param>
+            <when value="Variational-Bayes">
+                <expand macro="mcmc_options" />
```

Comment:
This one I'm not sure, are the mcmc options needed in variatianal-bayes or in metropolis-hastings?
You if condition below was indicating `#if $posteriorEstimationMethod.method != "Metropolis-Hastings"` but I'm not sure.

Reply:
MCMC options are **not** needed for Variational Bayes; **are** needed for the other two

Reply:
From the code it looks like the MCMC options are irrelevant if variational-bayes _is_ selected, so the if statement needs to be updated.

Reply:
Yes, it does need to be updated.